src/sql/workbench/services/profiler/common/interfaces.ts

```diff
@@ -147,8 +147,14 @@ export interface IProfilerViewTemplate {
 	columns: Array<IColumnViewTemplate>;
 }
 
+export enum EngineType {
+	SQLDB = 'SQLDB',
```

Comment:
we should verify this is consistent with various other code blocks using "Azure" or "AzureDB" or "IsCloud" etc.  It doesn't necessarily need to match, but we should make sure it makes sense overall.

Reply:
checked the docs page and its official name is Azure SQL Database, i am using AzureSQLDB for short here. but I am not planning to touch other places with this PR.